google/cloud/bigtable/examples/table_admin_snippets.cc

```diff
@@ -70,14 +42,11 @@ void CreateTable(google::cloud::bigtable::TableAdmin admin, int argc,
               << "\n";
   }
   //! [create table] [END bigtable_create_table]
-  (std::move(admin), table_id);
+  (std::move(admin), argv.at(0));
```

Comment:
Do we want `vector:at` in these cases?  That is, any run-time error generated here is really a static error in the "auto" call or the `MakeCommandEntry()` call, and a simple subscript would make that clearer.  (I don't recall seeing these in other instances.)

Reply:
I learned the hard way that it was very hard to debug mistakes in `RunAll()` with the number of arguments, the crash was very obscure without them.  I think we should change the existing examples to use `std::vector::at`, but if you think they are distracting then I can just use brackets for now. Just say the word.

Reply:
> I think we should change the existing examples to use `std::vector::at`

Sounds good to me.  Thanks.